clientupdate/distsign/distsign.go

```diff
@@ -0,0 +1,228 @@
+// Package distsign implements signature and validation of arbitrary
+// distributable files.
+//
+// There are 3 parties in this exchange:
+//   - builder, which creates files, signs them and publishes to server
+//   - server, which distributes signing keys, files and signatures
```

Comment:
Maybe specify public keys (to distinguish it from the secrets server)

Reply:
done